### DISCUSSION TITLE
要来一杯吗，prprprprpr

### DISCUSSION THREAD

**Discussion by justsmile3** [share121/inter-knot#2478] [其他]
纳西妲 ！              ![7%9H{(4VY6CJ4H%}R)1I1VR](https://github.com/user-attachments/assets/d5480cef-76bd-46cd-94a3-493c3b45f4c0)

**Comment by justsmile3** [comment]
![Uploading 1723651714872.gif…]()

**Comment by justsmile3** [comment]
<span style="color:red">我</span>
<span style="color:orange">永</span>
<span style="color:yellow">远</span>
<span style="color:green">喜</span>
<span style="color:blue">欢</span>
<span style="color:indigo">纳</span>
<span style="color:violet">西</span>
<span style="color:purple">妲</span>

**Comment by justsmile3** [comment]
**测试测试**

**Comment by nina-mmk** [comment]
有食物
![`OBEQOIVJ4M~LUXC%QOMX6Y](https://github.com/user-attachments/assets/9903ab9d-4797-4fa6-b9c2-b8a81320ef00)
![`OBEQOIVJ4M~LUXC%QOMX6Y](https://github.com/user-attachments/assets/35d1727d-7ade-4e45-a869-4f2048056ff5)
